src/components/collection/Selector/Table/Hydrator.tsx

```diff
@@ -35,20 +36,26 @@ function Hydrator({ selectedCollections }: Props) {
     } = useTableState('csl', publishedColumn, 'desc', tableRowsPerPage[0]);
 
     const query = useMemo(() => {
-        return getLiveSpecs_collectionsSelector(pagination, searchQuery, [
-            {
-                col: columnToSort,
-                direction: sortDirection,
-            },
-        ]);
-    }, [columnToSort, pagination, searchQuery, sortDirection]);
+        return getLiveSpecs_collectionsSelector(
+            pagination,
+            entity ?? 'collection',
+            searchQuery,
+            [
+                {
+                    col: columnToSort,
+                    direction: sortDirection,
+                },
+            ]
+        );
+    }, [columnToSort, entity, pagination, searchQuery, sortDirection]);
 
     const setDisabledRows = useStore(
         invariableStores['Collections-Selector-Table'],
```

Comment:
Yes... because when we reference the actual variable of `invariableStores` it can auto complete the string for us anyway. Hoping to slowly get away from need as many references to the store names enums.